doc/api/buffer.md

```diff
@@ -1907,6 +1907,9 @@ changes:
 Returns a new `Buffer` that references the same memory as the original, but
 offset and cropped by the `start` and `end` indices.
 
+Specifying 'end' greater than ['buf.length'] will return the same result as that of
```

Comment:
Looks good, but you should use ` instead of '
The resulting HTML looks different, and only ``[`bug.length`][]`` links to the right place - https://github.com/nodejs/node/pull/14720/files?short_path=c4f6105#diff-c4f6105249780774d5b3f276b4e4879f